src/main/java/com/zenuml/confluence/macro/SequenceMacro.java

```diff
@@ -65,9 +65,7 @@ public String execute(Map<String, String> map, String s, ConversionContext conve
         String outputType = conversionContext.getOutputType();
```

Comment:
It is not safe for this method to throw MacroExecutionException or RuntimeException up as there is not logic that we can control after this.
Wrap them all in try/catch and return a `<div> We are not able to render the macro. Please contact the support at https://zenuml.atlassian.net/servicedesk </div>`